gobblin-example/src/main/resources/avro-eventhub.job

```diff
@@ -3,13 +3,13 @@ gobblin.workDir=gobblin
 
 launcher.type=LOCAL
 
-
+# FIXME!
```

Comment:
Ah, this was because a converter class name changed and I wanted to update this file.